README.md

```diff
@@ -54,30 +54,49 @@ List all of the components
 ***
 
 ## 2) How we have structured our Backbone applications
-The point of backbone is to provide a very intentional structure to our front-end code, so that we know where to find the code that is handling the specific functionality of our application.
+The point of Backbone is to provide a very intentional structure to our front-end code, so that we know where to find the code that is handling the specific functionality of our application.
 
-With Backbone being a less opinionated library/framework, it also has less rules and conventions on
+With Backbone being a less opinionated library/framework, it also has less rules and conventions on how to organize your application and where you implement specific functionality.  
 
+We'll take a look at a sample Rolodex solution.  
 
-### App.js
+Fork & Clone this [repository.](https://github.com/Hamled/rolodex/tree/Hamled/impl).
```

Comment:
I think many of them have already done this and I think they can probably get away without putting the code on their computer, right?